bower.json

```diff
@@ -8,7 +8,7 @@
     "ember-qunit": "0.3.1",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.15",
-    "jquery": "^1.11.1",
+    "jquery": "1.11.1",
```

Comment:
Yeah, the Ember assert is removed so we can update ember itself in a separate PR.